### PR TITLE
add legal stuff

### PR DESCRIPTION
**Pull request by inkhey**
- add mit license
- add dco

### CODE REVIEW

File: DCO [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
```
Comment (tracim) [inline DCO]: You should put Algoo, here... no?
by the way, copyright should be 2013, 2018

Comment (inkhey) [reply DCO]: I don't think we should put it there, DCO v1.1 copyright owner are Linux Foundation.
Gitlab itself, let linux foundation copyright : https://gitlab.com/gitlab-org/dco/blob/master/README.md .

But you're right for date in MIT license.

Comment (tracim) [reply DCO]: ok, that's fine

**Comment by inkhey** [comment]
@tracim I do not really like to force an invalid extension in this type of file. LICENSE and DCO file contains plain text, not markdown.

**Pull request merged**